app/helpers/application_helper.rb

```diff
@@ -641,5 +632,30 @@ def list_to_string(list)
     list&.join(',') || ''
   end
 
+  def ontologies_selector(id:, label: nil, name: nil, selected: nil)
+    - get_ontologies_data
+    content_tag(:div) do
+      render(Input::SelectComponent.new(id: id, label: label, name: name, value: @onts_for_select, multiple: "multiple", selected: selected)) +
+      content_tag(:div, class: 'ontologies-selector-button', 'data-controller': 'ontologies-selector', 'data-ontologies-selector-id-value': id) do
+        content_tag(:div, t('ontologies_selector.clear_selection'), class: 'clear-selection', 'data-action': 'click->ontologies-selector#clear') +
+        link_to_modal(t('ontologies_selector.ontologies_advanced_selection'), "/ontologies_selector?id=#{id}", data: { show_modal_title_value: t('ontologies_selector.ontologies_advanced_selection')})
+      end
+    end
+  end
+
+  def save_and_cancel_buttons(save_html_options: nil , cancel_html_options: nil)
```

Comment:
do two helpers, one for save and the other for cancel. and check if they does not already exist somewhere

Reply:
These buttons just reuses the regular button component in a certain way related just to the ontologies selector and can't be reused somewhere else, and if we extract the params of it we'll end up just making a function that forwards the button component,
I think it's better to not extract it at all.